attachment_sync_gdrive/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# © 2018 Sunflower IT (http://sunflowerweb.nl)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Attachment Sync Google Drive",
+    "version": "11.0.1.0.0",
+    "category": "Storage",
+    "author": "Sunflower IT,Odoo Community Association (OCA)",
+    "website": "https://sunflowerweb.nl",
+    "license": "AGPL-3",
+    "summary": "Sync Attachments to Google Drive",
+    "depends": [
+        "structured_external_storage_all",
+        "google_drive",
+    ],
```

Comment:
The external python dependencies you mentioned in the README are missing here.

Strangely, they are also not in the [Odoo module](https://github.com/OCA/OCB/blob/11.0/addons/google_drive/__manifest__.py#L20)...